packages/koa-metrics/README.md

```diff
@@ -0,0 +1,74 @@
+# `@shopify/koa-metrics`
+
+Performance metrics implemented with DogStatsD (<https://docs.datadoghq.com/developers/dogstatsd/>) are used widely across services at Shopify. The standard metrics should be documented and following strong conventions.
+
+This package aims to provide standard middlewares and instrumentation tooling for metrics in Koa.
+
+## Installation
+
+```bash
+$ yarn add @shopify/koa-metrics
+```
+
+## Usage
+
+```javascript
+import Koa from 'koa';
+import metrics from '@shopify/koa-metrics';
+
+const app = new Koa();
+
+app.use(metrics(options));
```

Comment:
I always find it annoying when a usage example does not actually work as-is (options not defined)